src/RadixDlt.CoreApiSdk/Model/CreatedSubstate.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+
+namespace RadixDlt.CoreApiSdk.Model;
+
+public partial class StateUpdates
+{
+    public List<IAbc> MyCreated => CreatedSubstates.ToList<IAbc>();
+    public List<IAbc> MyUpdate => UpdatedSubstates.ToList<IAbc>();
+}
+
+public partial class CreatedSubstate : IAbc
+{
+    public SubstateValue PreviousValue => null;
+}
+
+public partial class UpdatedSubstate : IAbc
```

Comment:
I was initially thinking the same but it would only obscure the code. The reason I've introduced the interface is very NG-centric anyways as we always operate on "upserted substates + deleted substates" (see most recent changes).